pkg/drivers/mysql/mysql.go

```diff
@@ -24,7 +25,7 @@ const (
 
 var (
 	schema = []string{
-		`CREATE TABLE IF NOT EXISTS kine
```

Comment:
Could we perhaps leave the create table/database statements alone? I'm not sure if it's possible to not have access to the `information_schema` table, but if so leaving these statements as-is would allow for graceful fallback to the current behavior.